utils/datetime.ts

```diff
@@ -210,11 +210,11 @@ export function startOfMonth(d: Date, n = 0) {
 }
 
 export function endOfMonth(d: Date, n = 0) {
-  return new Date(+new Date(d.getFullYear(), d.getMonth() + 1 + n) - DAY);
+  return new Date(d.getFullYear(), d.getMonth() + 1 + n, 0);
```

Comment:
Oh, `0` in the 3rd arg means the last day of the previous month. I didn't know this